src/cliargs/core.lua

```diff
@@ -149,7 +149,8 @@ local function create_core()
   ---        The number of columns assigned to the argument descriptions, set to
   ---        0 to auto set the total width to 72.
   function cli:set_colsz(key_cols, desc_cols)
-    colsz = { key_cols or colsz[1], desc_cols or colsz[2] }
+    colsz[1] = key_cols or colsz[1]
```

Comment:
Interesting, now I read issue #55 and I get it. I'd rather we keep the current implementation and expose a getter for the printer to use at print time instead of passing it a reference and having to edit the table in-place.